packages/app-bridge/src/react/useDocumentPages.ts

```diff
@@ -24,17 +25,30 @@ export const useDocumentPages = (appBridge: AppBridgeTheme, documentId: number)
         setIsLoading(false);
     }, [appBridge, documentId]);
 
+    const refetchIfPageExistsInMap = useCallback(
```

Comment:
Why refetch on Targets event? Also if refetchIfPageExistsInMap is used only in `useEffect` put it there so you avoid usage of useCallback.

Reply:
Okay, you could do it this way. But it would be better for us to do a state update here of target ids without the additional call to API.

Reply:
will be adjusted in later pr when backend is updated 👍